src/services/doppler-legacy-client.ts

```diff
@@ -454,13 +499,13 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
     return mapHeaderDataJson(response.data);
   }
 
-  public async getAllPlans(): Promise<PlanModel[]> {
+  public async getAllPlans(): Promise<any> {
     const response = await this.axios.get('/WebApp/GetAllPlans');
     if (!response?.data) {
       throw new Error('Empty Doppler response');
     }
 
-    return mapPlanListJson(response.data);
+    return sanitizePlans(response.data.data).map(parsePlan).sort(compareByPlanType);
```

Comment:
I am not sure if it is necessary to sort now. 

When you deal with the plans later, you will probably "unsort" them and you have to sort them again. Maybe sorting now give us a false sense of sorting that could be error prone.

Reply:
I've tested with and without sorting, and either I sort here or move the sorting in the service, yes perhaps it is not as specific as it should be.

Reply:
In later stages, you have to sort anyway:

![image](https://user-images.githubusercontent.com/1157864/92940803-ef101f80-f425-11ea-9688-d5b9b2a7c291.png)

Reply:
Lo que quiero decir, es que tal vez lo más seguro sea asumir que los datos están desordenados, y ordenar recién cuando se necesita.

De otra manera, si ordenás los datos de entrada y asumís que siempre están ordenados, por ahí en una manipulación se desordenan (por ejemplo uniendo dos listas) y olvidás ordenarlos nuevamente.